src/EnergyPlus/ChillerElectricASHRAE205.hh

```diff
@@ -0,0 +1,170 @@
+        Real64 OilCoolerMassFlowRate{0};
+        PlantLocation OCPlantLoc{0, DataPlant::LoopSideLocation::Invalid, 0, 0};
+        int AuxiliaryHeatInletNode{0};
+        int AuxiliaryHeatOutletNode{0};
+        Real64 AuxiliaryVolFlowRate{0};
+        Real64 AuxiliaryMassFlowRate{0};
+        PlantLocation AHPlantLoc{0, DataPlant::LoopSideLocation::Invalid, 0, 0};
+        Real64 QOilCooler{0};
+        Real64 QAuxiliary{0};
+        Real64 OilCoolerEnergy{0};
+        Real64 AuxiliaryEnergy{0};
+
+        AmbientTempIndicator AmbientTempType{AmbientTempIndicator::Invalid};
+        int AmbientTempSchedule{0};       // Schedule index pointer
+        int AmbientTempZone{0};           // Number of ambient zone around tank
+        int AmbientTempOutsideAirNode{0}; // Number of outside air node
+        Real64 AmbientTemp{0};
+        Real64 AmbientZoneGain{0};         // Internal gain to zone from losses (W)
+        Real64 AmbientZoneGainEnergy{0};   // Internal gain to zone from losses (J)
+        std::string EndUseSubcategory{""}; // identifier use for the end use subcategory
+
+        // Default Constructor
+        ASHRAE205ChillerSpecs() = default;
+
+        void setOutputVariables(EnergyPlusData &state);
+
+        static ASHRAE205ChillerSpecs *factory(EnergyPlusData &state, std::string const &chillerName);
+
+        void simulate([[maybe_unused]] EnergyPlusData &state,
+                      const PlantLocation &calledFromLocation,
+                      bool FirstHVACIteration,
+                      Real64 &CurLoad,
+                      bool RunFlag) override;
+
+        void getDesignCapacities(
+            EnergyPlusData &state, const PlantLocation &calledFromLocation, Real64 &MaxLoad, Real64 &MinLoad, Real64 &OptLoad) override;
+
+        void initialize(EnergyPlusData &state, bool RunFlag, Real64 MyLoad) override;
+
+        void size(EnergyPlusData &state) override;
+
+        void findEvaporatorMassFlowRate(EnergyPlusData &state, Real64 &load, Real64 Cp);
+
+        Real64 findCapacityResidual(EnergyPlusData &, Real64 partLoadSequenceNumber, std::array<Real64, 4> const &par);
+
+        void calculate(EnergyPlusData &state,
+                       Real64 &MyLoad, // operating load
+                       bool RunFlag    // TRUE when chiller operating
+                       ) override;
+
+        void update(EnergyPlusData &state,
+                    Real64 MyLoad, // current load
+                    bool RunFlag   // TRUE if chiller operating
+                    ) override;
+
+        void oneTimeInit(EnergyPlusData &state) override;
+    };
+
+} // namespace ChillerElectricASHRAE205
+
+struct ChillerElectricASHRAE205Data : BaseGlobalStruct
+{
+    bool getInputFlag = true;
+    Array1D<ChillerElectricASHRAE205::ASHRAE205ChillerSpecs> Electric205Chiller;
+
+    void clear_state() override
+    {
+        *this = ChillerElectricASHRAE205Data();
+    }
+};
+
+} // namespace EnergyPlus
```

Comment:
:+1:

Reply:
I am surprised you didn't use an `EPVector` or a `std::vector` to hold the chillers.  That's another missed opportunity.

Reply:
This is pretty easy to change I imagine. I think the Array1D was just mimicked from the other chiller source files.

Reply:
It is a pretty easy change to make, but don't make it here.  I'm doing final testing locally, no need to waste CI cycles on this change. 

And yeah, it still shows up in some components, but as we refactor, we are trying hard to move to _at least_ `EPVector`, and ideally just `std::vector` for runtime-sized lists.  The change to `EPVector` would almost certainly require zero other changes.